userdocs/src/usage/nodegroup-additional-volume-mappings.md

```diff
@@ -0,0 +1,34 @@
+# Additional Volume Mappings
```

Comment:
does this need to be a new page? or can it be added to the existing page?

Reply:
We have custom DNS and upgrade and custom subnet separately. I thought it would be nice and searchable this way.

I can add it to `manage-nodegroups.md` if you think that's a better place for it. :)

Reply:
@aclevername I'm actually not sure what the 👍 was for in this case. Move it or leave it as is? :D